x-pack/legacy/plugins/apm/index.ts

```diff
@@ -11,10 +11,17 @@ import { APMPluginContract } from '../../../plugins/apm/server/plugin';
 import { LegacyPluginInitializer } from '../../../../src/legacy/types';
 import mappings from './mappings.json';
 import { makeApmUsageCollector } from './server/lib/apm_telemetry';
+import { initializeServiceMaps } from './server/lib/service_map/initialize_service_maps';
 
 export const apm: LegacyPluginInitializer = kibana => {
   return new kibana.Plugin({
-    require: ['kibana', 'elasticsearch', 'xpack_main', 'apm_oss'],
+    require: [
+      'kibana',
+      'elasticsearch',
+      'xpack_main',
+      'apm_oss',
+      'task_manager'
```

Comment:
This is another reason for not having `serviceMapEnabled` anymore: APM requires `task_manager` regardless.